app/api/health/apis/route.ts

```
@@ -0,0 +1,220 @@
+      try {
+        switch (apiName.toLowerCase()) {
+          case 'perplexity':
+            results.perplexity = await apiHealthService.checkPerplexityAPI();
+            break;
+          case 'delta':
+            results.delta = await apiHealthService.checkDeltaExchangeAPI();
+            break;
+          case 'coinmarketcap':
+          case 'cryptocompare':
+          case 'coinapi':
+            const cryptoAPIs = await apiHealthService.checkCryptoDataAPIs();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)